Sources/CryptorRSA/CryptorRSAKey.swift

```diff
@@ -318,28 +318,41 @@ extension CryptorRSA {
 				
 				throw Error(code: ERR_CREATE_CERT_FAILED, reason: "Unable to create certificate from certificate data.")
 			}
+			
+			var key: SecKey? = nil
+			#if swift(>=4.2)
```

Comment:
Why is this check needed?  I don't see the need for it...

Reply:
The travis without this was [failing](https://travis-ci.org/IBM-Swift/BlueRSA/jobs/551226377). I can't find it in the documentation but i believe this due to SecCertificateCopyKey only being introduced in Swift 4.2. Even with the `@available` tag the compiler fails since it doesn't know this function.